[DOC] Update example `docker run` commands + add submodule installation in README (#186)
neurobagel/bagel-cli: README.md

`singularity pull bagel.sif docker://neurobagel/bagelcli`

## Running the CLI
CLI commands can be accessed using the Docker/Singularity image.

> NOTE: The Docker examples below assume you are using the DockerHub image for the CLI. 

Reviewer: ```suggestion
NOTE: The Docker examples below assume that you are using the official neurobagel DockerHub image for the CLI.
```
better to be explicit

pip install --no-deps -e .
```

> Finally, we can install the `bids-examples` submodule used for testing:

Reviewer: ```suggestion
Finally, to run the test suite we need to install the `bids-examples` submodule:
```

```bash
git submodule init
git submodule update
```

> 

Reviewer: ```suggestion
Confirm that everything works well by running a test
`pytest .`
```
or something like that
